app/assets/javascripts/views/form_view.js

```diff
@@ -116,13 +116,13 @@ module.exports = Backbone.View.extend({
     return el;
   },
 
-  bindElementToAttribute: function(el, name, eventType) {
+  bindElementToAttribute: function(el, name, eventType, silent = true) {
     var that = this;
     eventType = typeof(eventType) != 'undefined' ? eventType : "change";
     $(el).on(eventType, function() {
       var obj = {};
       obj[name] = $(el).val();
-      that.model.set(obj, {silent: true});
+      that.model.set(obj, {silent: silent});
```

Comment:
Since the name of the variable is already `silent`, you can pass just: `that.model.set(obj, { silent });` and it'll convert it internally to `{ silent: silent }`